Sources/ClientRuntime/Retries/ExponentialBackoffStrategyOptions.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.TimeInterval
+
+struct ExponentialBackoffStrategyOptions {
```

Comment:
Configures the exponential backoff scheme.

Follows reference architecture, but some values are not yet used.

Reply:
Should we add a todo? (I am not sure if it is part of the upcoming PRs)

Reply:
I think I will keep the unused params in place, but mark them as unused, just to track differences between stated reference architecture & here.

(Neither reference architecture nor the SEP say what is to be done with backoff scale)